.github/workflows/test.yml

```diff
@@ -14,6 +14,7 @@ concurrency:
 
 env:
   DEFAULT_PYTHON: '3.13'
+  MIN_COVERAGE: 92
```

Comment:
Use latest coverage percent from current repo, not copy-paste along different repos